extensions/wikia/PortableInfobox/js/PortableInfobox.js

```diff
@@ -25,22 +25,46 @@
 	};
 
 	var CollapsibleGroup = {
-		init: function($content) {
+		init: function ($content) {
 			var $collapsibleGroups = $content.find('.pi-collapse');
 
-			$collapsibleGroups.each( function( index, group ) {
+			$collapsibleGroups.each(function (index, group) {
 				var $group = $collapsibleGroups.eq(index),
 					$header = $group.find('.pi-header');
 
-				$header.click( function() {
+				$header.click(function () {
 					$group.toggleClass('pi-collapse-closed');
 				});
 			});
 		}
 	};
 
-	mw.hook('wikipage.content').add(function($content) {
+	var CollapsibleData = {
+		init: function ($content) {
+			var $collapsibleDatas = $content.find('.pi-data.pi-collapse-data');
+
+			$collapsibleDatas.each(function () {
+				var $element = $(this);
+
+				// FIXME scrollHeight is calculated incorrectly so we need to subtract a magic number
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1134)